Reorganize the Schema Cache information into a separate reference page (#404)
PostgREST/postgrest-docs: admin.rst

    docker run -v "$(pwd)/init.sh":"/docker-entrypoint-initdb.d/init.sh" -d postgres
    docker logs -f <container-id>

Schema Reloading
----------------

> If the schema changes while the server is running it results in a stale cache and leads to errors. To learn how to refresh the cache see :ref:`schema_reloading`.

Reviewer: ```suggestion
Changing the schema while the server is running can lead to errors due to a stale schema cache. To learn how to refresh the cache see :ref:`schema_reloading`.
```
